components/icon/tests/unit/icons.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import { GEL } from '@westpac/core';
+import { render } from '@testing-library/react';
+import wbc from '@westpac/wbc';
+import { overridesTest } from '../../../../helpers/tests/overrides-test.js';
+import { nestingTest } from '../../../../helpers/tests/nesting-test.js';
+import * as icons from '@westpac/icon';
+import { Icon } from '../../src/Icon.js'; // needs to be imported this way as it is not included in the Icon library
+import { blenderIcon } from '../../src/overrides/icon.js';
+
+overridesTest({
+	name: 'icon',
+	overrides: ['Icon'],
+	Component: (props: any) => <Icon copyrightYear="2020" size="medium" {...props} />,
+});
+
+nestingTest({
+	name: 'icon',
+	Component: (props: any) => <Icon {...props} />,
+});
+
+function SimpleIcon(props: any) {
```

Comment:
Would it be possible to define the component contract instead of using "any"?

Reply:
Just leaving a comment here after discussing through teams with @samithaf and @soobin93. The Icon component is not a TS component and is still JS so there is no interface to import and can be used would be better to convert Icon to TS but that should be a separate task. I have updated the Label tests using the interface exported from that component and we will be going forward doing that for future TS tests now we have a working example.

Reply:
Well explained. The PR is looking good.